src/main/java/jsr223/docker/file/DockerFileScriptEngine.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * ProActive Parallel Suite(TM):
+ * The Open Source library for parallel and distributed
+ * Workflows & Scheduling, Orchestration, Cloud Automation
+ * and Big Data Analysis on Enterprise Grids & Clouds.
+ *
+ * Copyright (c) 2007 - 2017 ActiveEon
+ * Contact: contact@activeeon.com
+ *
+ * This library is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License
+ * as published by the Free Software Foundation: version 3 of
+ * the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * If needed, contact us to obtain a release under GPL Version 2 or 3
+ * or a different license than the AGPL.
+ */
+package jsr223.docker.file;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.Reader;
+import java.io.StringWriter;
+import java.util.Map;
+
+import javax.script.*;
+
+import jsr223.docker.compose.bindings.MapBindingsAdder;
+import jsr223.docker.compose.bindings.StringBindingsAdder;
+import jsr223.docker.compose.file.write.ConfigurationFileWriter;
+import jsr223.docker.compose.utils.Log4jConfigurationLoader;
+import jsr223.docker.compose.utils.ScriptContextBindingsExtractor;
+import jsr223.docker.compose.yaml.VariablesReplacer;
+import jsr223.docker.file.DockerFileCommandCreator;
+import jsr223.docker.file.DockerFileScriptEngine;
+import jsr223.docker.file.DockerFileScriptEngineFactory;
+import lombok.extern.log4j.Log4j;
+import processbuilder.SingletonProcessBuilderFactory;
+import processbuilder.utils.ProcessBuilderUtilities;
+
+
+@Log4j
+public class DockerFileScriptEngine extends AbstractScriptEngine {
+
+    private static final String DOCKER_HOST_PROPERTY_NAME = "DOCKER_HOST";
```

Comment:
Did you have a look what this is in the docker-compose script engine. Not sure if we need it here.